src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -242,15 +259,19 @@ class Force<ExecutionSpace, ForceModel<PMB, Fracture>>
         Kokkos::RangePolicy<exec_space> policy( 0, n_local );
         Kokkos::parallel_for( "CabanaPD::ForcePMBDamage::computeFull", policy,
                               force_full );
+
+        _timer.stop();
     }
 
     template <class PosType, class WType, class DamageType, class ParticleType,
               class NeighListType, class MuView, class ParallelType>
     double computeEnergyFull( WType& W, const PosType& x, const PosType& u,
                               DamageType& phi, const ParticleType& particles,
                               const NeighListType& neigh_list, MuView& mu,
-                              const int n_local, ParallelType& ) const
+                              const int n_local, ParallelType& )
     {
+        _energy_timer.stop();
```

Comment:
@streeve Should this be " _energy_timer.start();"? We have "_energy_timer.stop();" in Line 311.

Reply:
Yes, thank you